Introduction.playground/Pages/6. Memory Management - Part Two.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,53 @@
+//: [Previous](@previous)
+
+import Foundation
+import RxSwift
+
+//: # Memory Management - Part Two
+//:
+//: In previous example we learned that certain sequence observers are automatically disposed when a sequence completes.
```

Comment:
It's just a case of wording, but I think it's important to put emphasis on the fact that *every* subscription *always* disposes when *any* observable emits onCompleted on onError. It's not just sequence observers, it's one of the invariants of the Rx framework